recipes-kernel/linux/files/ateccgen2.cfg

```diff
@@ -14,3 +14,8 @@ CONFIG_BMP280=y
 CONFIG_BMP280_I2C=y
 CONFIG_BMP280_SPI=y
 CONFIG_CIFS=m
+# CONFIG_WLAN is not set
+# CONFIG_BT is not set
+# CONFIG_RFKILL is not set
+# CONFIG_CFG80211 is not set
+# CONFIG_CAN is not set
```

Comment:
For some reason, removing CONFIG_SOUND from defconfig will cause some issues when compiling gpio-switch.c, in a similar fashion as in:
https://www.mail-archive.com/linux-kernel@vger.kernel.org/msg2541557.html

Temporarily removed omission of CONFIG_SOUND, but note that RCU image does not require it.